app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -41,16 +54,88 @@ def get_jwt_identity
         @authentication_parameters.jwt_identity =  @jwt_configuration.jwt_identity(@authentication_parameters)
       end
 
+      def validate_user_has_access_to_webservice
+        @validate_role_can_access_webservice.(
+          webservice: webservice,
+            account: account,
+            user_id: @authentication_parameters.jwt_identity,
+            privilege: 'authenticate'
```

Comment:
Do we have it as a const somewhere?

Reply:
I copied pasta it from here 
https://github.com/cyberark/conjur/blob/6f45b33c55d6efcef791d71155d09592a9e1f5c2/app/domain/authentication/authenticate.rb#L63

Reply:
Moved to const only in authn-jwt code